lib/client/api.go

```diff
@@ -2797,6 +2797,13 @@ func (tc *TeleportClient) connectToProxy(ctx context.Context) (*ProxyClient, err
 	return pc, nil
 }
 
+// confirmSSHConnectivityErrorMsg returns the error message for Proxy SSH
+// connectivity errors.
+func confirmSSHConnectivityErrorMsg(proxyAddress string) string {
+	return fmt.Sprintf("Unable to connect to ssh proxy address=%v required for client interactions. "+
```

Comment:
This is written like a sentence, but `=` is more suited for logs than plain english sentences.

Suggested a small adjustment
```suggestion
	return fmt.Sprintf("Unable to connect to ssh proxy at %v. "+
```